not-supported/backup-viewer.sh

```diff
@@ -48,10 +48,18 @@ fi
 # MC is needed for this
 if ! is_this_installed mc
 then
-    msg_box "For viewing backups we will need Midnight Commander, which is a command line file explorer.
-Please install it before you can continue with this script by running:
-'sudo bash /var/scripts/menu.sh' choose 'Main Menu => Additional Apps'"
-    exit 1
+    msg_box "For viewing backups we will need Midnight Commander, which is a command line file explorer."
```

Comment:
Well, you could view them with Webmin as well which is more user friendly IMHO.

Reply:
View them from the command line via webmin?
Actually, I recommend to do it like this:
https://szaimen.github.io/Nextcloud-NAS-Guide/docs/restore-files

Reply:
Midnight commander is useful to navigate through the folders and see if all files are backed up, so a very simple and fast way to check your backups. Also as long as midnight commander is open, the backup is mounted. Afterwards it gets directly unmounted, etc.

Reply:
For a windows user seeing MC, it will be like :exploding_head: :)

What I mean is that you have a file manager in Webmin as well.

Reply:
I see you haven't used it with my script before ;)
first I'll give some advices how to use it:
![image](https://user-images.githubusercontent.com/42591237/113486787-af3d8200-94b4-11eb-9429-b1402bb1c984.png)
And after you confirm that you've understood and remember those rules, mc gets opened with the correct directory and looks like this:
![image](https://user-images.githubusercontent.com/42591237/113486816-cf6d4100-94b4-11eb-9743-5227eb69531b.png)

I think this is completely doable and manageable for any user 😉

Reply:
Of course copy and move doesn't work as easily with MC but for that I recommend this https://szaimen.github.io/Nextcloud-NAS-Guide/docs/restore-files

Reply:
The same is of course possible with wrbmin but there is no way we can automate webmin to open the correct directory, etc.

Reply:
It looks like the directory is accessible by opening `https://192.168.178.86:10000/filemin/index.cgi?path=/tmp/borg` in a browser but before you can do this you must be logged in to webmin so it is really hard to automate this... Also it doesn't seem to support any normal mime type like pictures or movies. It only supports viewing text files as far as I can see...

Reply:
So how to proceed from here? I don't want to make webmin a requirement of this script because you can also use Remotedesktop for this which is much better, imo but won't work in all cases, of course...

Reply:
@enoch85 I hope that I fixed this review with 0d0e808

Reply:
I've made it possible now to choose between different options. MC, Webmin and Remotedesktop are now possible. I hope this finally addresses this review perfectly.